### PR TITLE
Add hermit cli

### PR DESCRIPTION
**Pull request by marcusolsson**
Introduces a CLI client for querying the Eremetic server.

### CODE REVIEW

**Comment by alde** [comment]
Should add to the README that there is a cli tool, and link to the cli README

**Comment by marcusolsson** [comment]
Good point. Will add it.

File: cmd/hermit/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Fetch information about a specific task.
+    
+    hermit task eremetic-task-id-abc123
+
+Fetch the logs of a task.
+
+    hermit logs -file stderr eremetic-task-id-abc123
+
+## Configuration
```
Comment (alde) [inline cmd/hermit/README.md]: Are these a .hermit config file, ENV vars, flags?

File: cmd/hermit/main.go [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+		}
+		httpClient.Transport = tr
+	}
+
+	ec, err := client.New(srv, httpClient)
+	if err != nil {
+		exitWithError(err)
+	}
+
+	if runCmd.Parsed() {
```
Comment (alde) [inline cmd/hermit/main.go]: Could these be broken into their own functions to make the main func a bit shorter and easier to read?

**Comment by alde** [comment]
lgtm

**Pull request merged**